discord/flags.py

```diff
@@ -2397,3 +2398,59 @@ def content_inventory_entry(self):
         longer displayed.
         """
         return 1 << 5
+
+
+class GuildInviteFlags(BaseFlags):
```

Comment:
```suggestion
class InviteFlags(BaseFlags):
```
kinda unnecessary imo